packages/perseus-core/src/utils/generators/group-widget-generator.test.ts

```diff
@@ -0,0 +1,100 @@
+import {
+    generateGroupOptions,
+    generateGroupWidget,
+} from "./group-widget-generator";
+import {generateRadioWidget} from "./radio-widget-generator";
+
+import type {GroupWidget, PerseusGroupWidgetOptions} from "../../data-schema";
+
+describe("generateGroupOptions", () => {
+    it("builds a default group options object", () => {
+        // Arrange, Act
+        const options: PerseusGroupWidgetOptions = generateGroupOptions();
```

Comment:
Do you have to explicitly provide the type if the function already has a defined return type?

Reply:
Nope. Sorry for the inconsistency between different widget generator tests.